godot-codegen/src/util.rs

```diff
@@ -670,11 +686,28 @@ fn to_rust_type_uncached(full_ty: &GodotTy, ctx: &mut Context) -> RustTy {
         };
     }
 
-    let qualified_enum = ty
-        .strip_prefix("enum::")
-        .or_else(|| ty.strip_prefix("bitfield::"));
+    if let Some(bitfield) = ty.strip_prefix("bitfield::") {
+        return if let Some((class, enum_)) = bitfield.split_once('.') {
+            // Class-local bitfield
```

Comment:
```suggestion
            // Class-local bitfield.
```

Reply:
I should probably also change this for the enum also 😅